lib/screens_web/router.ex

```diff
@@ -9,6 +9,12 @@ defmodule ScreensWeb.Router do
     plug :protect_from_forgery
   end
 
+  pipeline :browser_no_csrf do
+    plug :accepts, ["html"]
+    plug :fetch_session
+    plug :fetch_flash
+  end
```

Comment:
Would it make sense to build the `:browser` pipeline on top of this one, since they share everything but CSRF? I wasn't sure how to do this elegantly...

Reply:
Yes, that makes sense! I'd probably do:

```
  pipeline :browser do
    plug :accepts, ["html"]
    plug :fetch_session
    plug :fetch_flash
  end

  pipeline :csrf_protection do
    plug :protect_from_forgery
  end
```

and then for the appropriate scopes, you'd do
- `pipe_through [:redirect_prod_http, :browser]` or
- `pipe_through [:redirect_prod_http, :browser, :csrf_protection]`

Reply:
This makes sense but I'm a little worried that it now requires extra work to do the default thing. "Secure by default" and all that.

How about this?
```ex
pipeline :browser_no_csrf do
  plug :accepts, ["html"]
  plug :fetch_session
  plug :fetch_flash
end

pipeline :browser do
  plug :browser_no_csrf
  plug :protect_from_forgery
end
```
I just checked, and you can reuse one pipeline inside another without issues!

Reply:
Sure! Works for me